src/main/java/simplediff/DiffController.java

```diff
@@ -1,63 +1,96 @@
 package simplediff;
 
 import java.io.IOException;
-
-import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
-
 import simplediff.gumtree.diff.web.WebDiff;
 
 @RestController
 public class DiffController {
 
+  private static final String BASH_PATH;
+  private static final String CLEANUP_COMMAND;
+  private static final String ORIGINAL_SOURCE_FOLDER = "source";
+  private static final String MODIFIED_SOURCE_FOLDER = "target";
+
+  static {
+    final String nameOS = System.getProperty("os.name");
+    if (nameOS.equals("Windows 10")) {
+      BASH_PATH = "C:\\Program Files\\Git\\git-bash.exe";
+    } else {
+      BASH_PATH = "bash";
+    }
+    CLEANUP_COMMAND =
+        String.format("rm -rf %s && rm -rf %s", ORIGINAL_SOURCE_FOLDER, MODIFIED_SOURCE_FOLDER);
```

Comment:
Probably okay for our purposes, but we should be aware that this kind of stuff means we won't be able to support paths with spaces